.eslintrc.json

```diff
@@ -1,33 +1,31 @@
 {
-    "env": {
-        "browser": true,
-        "es2021": true,
-        "jest": true
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "react-app",
+    "react-app/jest",
+    "airbnb",
+    "plugin:prettier/recommended"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
     },
-    "extends": [
-       "react-app",
-       "react-app/jest",
-       "airbnb",
-       "plugin:prettier/recommended"
-    ],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "plugins": [
-        "react",
-        "prettier"
-    ],
-    "rules": {
-        "quotes":["error","single",{"avoidEscape": true}],
-        "react/jsx-uses-react":["off"],
-        "react/react-in-jsx-scope":["off"],
-        "react/jsx-props-no-spreading":["warn"],
-        "no-param-reassign":["off"],
-        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-        "testing-library/prefer-screen-queries":["off"]
-    }
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["react", "prettier"],
+  "rules": {
+    "quotes": ["error", "single", { "avoidEscape": true }],
+    "react/jsx-uses-react": ["off"],
+    "react/react-in-jsx-scope": ["off"],
+    "react/jsx-props-no-spreading": ["warn"],
+    "no-param-reassign": ["off"],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "testing-library/prefer-screen-queries": ["off"],
+    "import/prefer-default-export": 0
```

Comment:
Please don't touch the eslintrc if we absolutely dont need to. It helps us write production-level code and we must follow its rules, hence do not add exceptions to it like this unless we absolutely have to.